.travis.yml

```diff
@@ -10,6 +10,10 @@ virtualenv:
   system_site_packages: true
 
 before_install:
+<<<<<<< HEAD
+  - pip install mpmath
```

Comment:
Ohh Yes. I'll remember that from next time on-wards. 

Should I remove this HEAD and MASTER line in this file, and then push a re-commit? Or what should I do here?